### ISSUE TITLE
Continuous integration build timeouts

### ISSUE THREAD

**Issue by jugglinmike** [w3c/web-platform-tests#6202]
As of 10:00 EST on Friday, June 9, all TravisCI jobs that rely on automating the Firefox web browser began to time out. Although [a new version of the GeckoDriver utility (0.17.0) was published today](https://github.com/mozilla/geckodriver/releases/tag/v0.17.0), this timeout also occurs when using the prior release (0.16.1). Using version 54 of Firefox (as opposed to the "Nightly" version 55) corrects the problem. This appears to be caused by a bug in the latest Nightly release of Firefox itself.

Because pull requests are gated on positive test results in this project, the timeout is interrupting ongoing work. I am experimenting with a patch that will instead use the "Beta" release channel. If that resolves the problem, I plan to merge it to `master` as a short-term solution until the underlying WebDriver issue is addressed.

(Note that the use of Firefox's Beta channel may also be generally preferable moving forward. It will help shield this project from similar regressions. Additionally, Firefox Beta channel's release cycle is analogous to the "dev"/"unstable" channel that is currently in use for the Chrome browser.)

**Comment by jugglinmike** [comment]
The patch to switch the the "Beta" release channel is available at https://github.com/w3c/web-platform-tests/pull/6204. While this addresses the timeout as intended, unrelated differences between the releases cause new failures. See the comment thread of that pull request for more information.

**Comment by jugglinmike** [comment]
[I reported the problem to the Firefox team](https://bugzilla.mozilla.org/show_bug.cgi?id=1371868) and learned that it had already been identified. It was resolved in the very next release of Firefox Nightly: https://bugzilla.mozilla.org/show_bug.cgi?id=1371319

The more general risk of instability from Firefox's Nightly release channel remains. In gh-6204, we've decided that the frequency of problems this causes is acceptable for the advantages of testing the very latest build (see that discussion for more information).
